### PR TITLE
Basic motion tracking feature

### PR DESCRIPTION
**Pull request by grahamhome**
Supports throwing the device into the air and seeing a score of 0-10 corresponding to the distance thrown.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,6 +65,17 @@
 		<button onclick="toggleSensors()" class="charcoal">Toggle sensors</button>
 	</p>
 
+	<!--Temporary output element-->
+	<!--<h2>Console:</h2>
+	<p>
+		<span id="TestConsole">[Waiting for value]</span>
```
Comment (cman131) [inline index.html]: Should this test console be left in here?

File: index.html [hunk 2]
```diff
@@ -99,35 +104,92 @@
...
+	var peak_level = 0.05 //0.01 // Upper bound for velocity change indicating apogee reached (small)
+
+	var v = [0] 	// Velocity change measurements (initially 0)
+
+	var time_s = 0 // Start time variable
+	var time_t = 0 // Throw time variable
+	var time_a = 0 // Apogee time variable 
+	var time_c = 0 // Catch time variable
+
+	var n_t = 0
```
Comment (cman131) [inline index.html]: What are these values representing?

File: index.html [hunk 3]
```diff
@@ -244,43 +271,157 @@
...
+		//displayValue('TestConsole', "Falling time calculation: "+t+" ms "+"Height: "+df+"\' ' "+di+"\""+"\nYour Score: "+s)
+		displayValue("HeightField", s)
+
+		// Display important info on computer for analysis
+		/*console.log("Velocity changes: " + v)
+		console.log("Time_t: "+time_t)
+		console.log("Time_a: "+time_a)
+		console.log("Time_c: "+time_c)*/
+
+		v = [0]
```
Comment (cman131) [inline index.html]: I would like to see a comment mentioning that you are reseting these values

File: index.html [hunk 4]
```diff
@@ -244,43 +271,157 @@
...
+
+			//TODO: Count number of throw, catch, and apogee events.
+
+			// When thrown:
+			if (caught && (vChange >= throw_level)) {
+				time_s = (new Date()).getTime()
+				time_t = (new Date()).getTime() - time_s
+				caught = false
+				peaked = false
+				displayValue("TestConsole", "Thrown")
```
Comment (cman131) [inline index.html]: Again, I believe the test console should be left out

File: index.html [hunk 5]
```diff
@@ -244,43 +271,157 @@
...
+				displayValue("TestConsole", "Peaked")
+
+			//When caught (and apogee detected):
+			} else if (!caught && peaked && (vChange >= catch_level)) {
+				time_c = (new Date()).getTime() - time_s
+				caught = true
+				displayValue("TestConsole", "Caught")
+				toggleTracking() //Don't wait for a button press
+
+			/*
```
Comment (cman131) [inline index.html]: if the code is no longer needed I recommend deleting it

File: index.html [hunk 6]
```diff
@@ -351,8 +469,29 @@
 
 		// Update the value displayed.
 		displayValue('HumidityData', string)
+		// Update the value displayed.
+		var string = 'raw: 0x' + bufferToHexStr(data, 0, 1)
+		displayValue('KeypressData', string)
 	}
 
+	/*function accelerometerHandler(data)
```
Comment (cman131) [inline index.html]: Unneeded code

**Comment by alexgerst** [comment]
@grahamhome @cman131 What is the status of this PR? Can we get these comments resolved and try to merge today?

**Comment by alexgerst** [comment]
Delaying PR until spin data is implemented

**Pull request closed without merging**